styles/styles.css

```diff
@@ -0,0 +1,253 @@
+
+/*Universal Stylings*/
+
+* {
+  box-sizing: border-box;
+  font-family: 'Roboto Slab', serif;
+}
+
+/*Header*/
+
+.header {
+  background-image: url("../images/header-image.jpg");
+  height: 100%;
+  width: 100%;
+  color: white;
+  display: flex;
+  flex-direction: row;
+  align-content: space-between;
+  justify-content: space-between;
+}
+
+.title {
+  display: flex;
+  flex-direction: column;
+  align-content: flex-start;
+  justify-content: center;
+  margin-left: 5em;
+  margin-top: 2em;
+  margin-bottom: 5em;
+}
+
+.title > h1 {
+  font-style: italic;
+  font-size: 1.4em;
+  color: #4CAF50;
+}
+
+.title > img {
+  width: 100%;
+
+}
+
+.checkout {
+  display: flex;
+  flex-direction: column;
+  font-weight: bold;
+  float: right;
+
+}
+
+.checkout > a {
+  text-decoration: none;
+  color: #ffffff;
+  background-color: #4CAF50;
+  width: 40%;
+}
+
+.cart {
+  font-size: 1em;
+  color: white;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+}
+
+.cart > img {
+  width: 10%;
+}
+
+/* Header: Naviagation */
+
+.top-navigation {
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+  padding-left: 15%;
+  width: 100%;
+  background-color: #D0D0D0;
+  display: flex;
+  align-items: center;
+  margin-bottom: 2.5rem;
+
+}
+
+.top-navigation > a{
+  text-decoration: none;
+  color: #1c356d;
+  font-weight: bold;
+  padding-left: 2em;
+}
+
+
+
+
+/*Body*/
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-left: 5rem;
+
+}
+
+/*Body: Doggo grid*/
+
+
+.clean-text {
+  text-decoration: none;
+  text-align: left;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 3%;
+  margin-bottom: 10%;
+}
+
+.doggo-grid {
+  border: 1px solid #e8e8e8;
+}
+.doggo-grid > img {
+  width: 100%;
+  height: auto;
+}
+.doggo-grid > h3 {
```

Comment:
Make sure all of your css is aligned and spaced out right. Consider it "good penmanship"

```
.doggo-grid > h3 {
  color: #1c356d;
  padding: 0 5%;
}

.doggo-grid > p {
  line-height: 1.5;
  padding: 0 5%;
}
```